Xamarin.Forms.Platform.Android/AppCompat/FrameRenderer.cs

```diff
@@ -286,5 +290,22 @@ void UpdateShadow()
 			else
 				CardElevation = 0f;
 		}
+
+		void UpdateCornerRadius()
+		{
+			if (_defaultCornerRadius == -1f)
+			{
+				_defaultCornerRadius = Radius;
+			}
+
+			float cornerRadius = Element.CornerRadius;
+
+			if (cornerRadius == -1f)
```

Comment:
@StephaneDelcroix I just checked, on Android for example a negative value doesn't produce any error, the corners are drawn like with 0 radius. It's up to the platform to interpret it.